nextjs-client/pages/index.js

```diff
@@ -11,5 +13,18 @@ export default function Home() {
 }
 
 Home.getLayout = function getLayout(page) {
-  return <Layout currentPage="home">{page}</Layout>;
+  return <Layout currentPage='home'>{page}</Layout>;
 };
+
+export async function getStaticProps() {
+  const yogaClassData = await client.fetch(yogaClassesQuery);
+
+  const data = { yogaClassData };
```

Comment:
`data = {yogaClassData}` data here is the same as the Home page received as props? Wasn't sure about the data that the Home page's prop `data`.

Reply:
correct. you will access it with `props.data.yogaClassData` on the index page.